File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/obstetrics/gymissabort/d/GyMissAbortRecDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using System.Collections.Generic;

namespace iih.ci.mr.nu.obstetrics.gymissabort.d
{
    /// <summary>
    /// GyMissAbortRecDO 的摘要说明。
    /// </summary>
    public class GyMissAbortRecDO : BaseDO {

		public const string TABLE_NAME = "CI_MR_NU_GYMISSABORTREC";
		public const string TABLE_ALIAS_NAME = "a1";

        public GyMissAbortRecDO() {
        }
		public string Id_rec {
            get { return getAttrVal<string>("Id_rec",null); }
            set { setAttrVal<string>("Id_rec", value); }
        }
		public string Id_ass {
            get { return getAttrVal<string>("Id_ass",null); }
            set { setAttrVal<string>("Id_ass", value); }
        }
		public DateTime? Dt_rec {
            get { return getAttrVal<FDateTime>("Dt_rec",null); }
            set { setAttrVal<FDateTime>("Dt_rec", value); }
        }
		public FDouble Temp {
            get { return getAttrVal<FDouble>("Temp",null); }
            set { setAttrVal<FDouble>("Temp", value); }
        }
		public int? Pulse {
            get { return getAttrVal<int?>("Pulse",null); }
            set { setAttrVal<int?>("Pulse", value); }
        }
		public int? Breath {
            get { return getAttrVal<int?>("Breath",null); }
            set { setAttrVal<int?>("Breath", value); }
        }
		public int? Sbp {
            get { return getAttrVal<int?>("Sbp",null); }
            set { setAttrVal<int?>("Sbp", value); }
        }
		public int? Dbp {
            get { return getAttrVal<int?>("Dbp",null); }
            set { setAttrVal<int?>("Dbp", value); }
        }
		public FDouble Spo2 {
            get { return getAttrVal<FDouble>("Spo2",null); }
            set { setAttrVal<FDouble>("Spo2", value); }
        }
		public string Id_bleed {
            get { return getAttrVal<string>("Id_bleed",null); }
            set { setAttrVal<string>("Id_bleed", value); }
        }
		public string Sd_bleed {
            get { return getAttrVal<string>("Sd_bleed",null); }
            set { setAttrVal<string>("Sd_bleed", value); }
        }
		public int? Urinevolume {
            get { return getAttrVal<int?>("Urinevolume",null); }
            set { setAttrVal<int?>("Urinevolume", value); }
        }
		public int? Drainage {
            get { return getAttrVal<int?>("Drainage",null); }
            set { setAttrVal<int?>("Drainage", value); }
        }
		public string Obserilness {
            get { return getAttrVal<string>("Obserilness",null); }
            set { setAttrVal<string>("Obserilness", value); }
        }
		public string Id_rec_psn {
            get { return getAttrVal<string>("Id_rec_psn",null); }
            set { setAttrVal<string>("Id_rec_psn", value); }
        }
		public string Createdby {
            get { return getAttrVal<string>("Createdby",null); }
            set { setAttrVal<string>("Createdby", value); }
        }
		public DateTime? Createdtime {
            get { return getAttrVal<FDateTime>("Createdtime",null); }
            set { setAttrVal<FDateTime>("Createdtime", value); }
        }
		public string Modifiedby {
            get { return getAttrVal<string>("Modifiedby",null); }
            set { setAttrVal<string>("Modifiedby", value); }
        }
		public DateTime? Modifiedtime {
            get { return getAttrVal<FDateTime>("Modifiedtime",null); }
            set { setAttrVal<FDateTime>("Modifiedtime", value); }
        }
		public string Name_bleed {
            get { return getAttrVal<string>("Name_bleed",null); }
            set { setAttrVal<string>("Name_bleed", value); }
        }
		public string Name_rec_psn {
            get { return getAttrVal<string>("Name_rec_psn",null); }
            set { setAttrVal<string>("Name_rec_psn", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "CI_MR_NU_GYMISSABORTREC";
        }
        
        /// <summary>
        /// 返回表别名
        /// </summary>
        /// <returns></returns>
        public override string getAliasTableName()
        {
            return "a1";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id_rec";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.obstetrics.gymissabort.d.GyMissAbortRecDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id_rec", "Id_ass", "Dt_rec", "Temp", "Pulse", "Breath", "Sbp", "Dbp", "Spo2", "Id_bleed", "Sd_bleed", "Urinevolume", "Drainage", "Obserilness", "Id_rec_psn", "Createdby", "Createdtime", "Modifiedby", "Modifiedtime", "Name_bleed", "Name_rec_psn"};
        }
        
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/obstetrics/gymissabort/d/GyMissAbortRecDO.java
package iih.ci.mr.nu.obstetrics.gymissabort.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.mr.nu.obstetrics.gymissabort.d.desc.GyMissAbortRecDODesc;
import java.math.BigDecimal;

/**
 * 护理观察记录 DO数据 
 * 
 */
public class GyMissAbortRecDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID_REC= "Id_rec";
	public static final String ID_ASS= "Id_ass";
	public static final String DT_REC= "Dt_rec";
	public static final String TEMP= "Temp";
	public static final String PULSE= "Pulse";
	public static final String BREATH= "Breath";
	public static final String SBP= "Sbp";
	public static final String DBP= "Dbp";
	public static final String SPO2= "Spo2";
	public static final String ID_BLEED= "Id_bleed";
	public static final String SD_BLEED= "Sd_bleed";
	public static final String URINEVOLUME= "Urinevolume";
	public static final String DRAINAGE= "Drainage";
	public static final String OBSERILNESS= "Obserilness";
	public static final String ID_REC_PSN= "Id_rec_psn";
	public static final String CREATEDBY= "Createdby";
	public static final String CREATEDTIME= "Createdtime";
	public static final String MODIFIEDBY= "Modifiedby";
	public static final String MODIFIEDTIME= "Modifiedtime";
	public static final String NAME_BLEED= "Name_bleed";
	public static final String NAME_REC_PSN= "Name_rec_psn";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId_rec() {
		return ((String) getAttrVal("Id_rec"));
	}	
	public void setId_rec(String Id_rec) {
		setAttrVal("Id_rec", Id_rec);
	}
	public String getId_ass() {
		return ((String) getAttrVal("Id_ass"));
	}	
	public void setId_ass(String Id_ass) {
		setAttrVal("Id_ass", Id_ass);
	}
	public FDateTime getDt_rec() {
		return ((FDateTime) getAttrVal("Dt_rec"));
	}	
	public void setDt_rec(FDateTime Dt_rec) {
		setAttrVal("Dt_rec", Dt_rec);
	}
	public FDouble getTemp() {
		return ((FDouble) getAttrVal("Temp"));
	}	
	public void setTemp(FDouble Temp) {
		setAttrVal("Temp", Temp);
	}
	public Integer getPulse() {
		return ((Integer) getAttrVal("Pulse"));
	}	
	public void setPulse(Integer Pulse) {
		setAttrVal("Pulse", Pulse);
	}
	public Integer getBreath() {
		return ((Integer) getAttrVal("Breath"));
	}	
	public void setBreath(Integer Breath) {
		setAttrVal("Breath", Breath);
	}
	public Integer getSbp() {
		return ((Integer) getAttrVal("Sbp"));
	}	
	public void setSbp(Integer Sbp) {
		setAttrVal("Sbp", Sbp);
	}
	public Integer getDbp() {
		return ((Integer) getAttrVal("Dbp"));
	}	
	public void setDbp(Integer Dbp) {
		setAttrVal("Dbp", Dbp);
	}
	public FDouble getSpo2() {
		return ((FDouble) getAttrVal("Spo2"));
	}	
	public void setSpo2(FDouble Spo2) {
		setAttrVal("Spo2", Spo2);
	}
	public String getId_bleed() {
		return ((String) getAttrVal("Id_bleed"));
	}	
	public void setId_bleed(String Id_bleed) {
		setAttrVal("Id_bleed", Id_bleed);
	}
	public String getSd_bleed() {
		return ((String) getAttrVal("Sd_bleed"));
	}	
	public void setSd_bleed(String Sd_bleed) {
		setAttrVal("Sd_bleed", Sd_bleed);
	}
	public Integer getUrinevolume() {
		return ((Integer) getAttrVal("Urinevolume"));
	}	
	public void setUrinevolume(Integer Urinevolume) {
		setAttrVal("Urinevolume", Urinevolume);
	}
	public Integer getDrainage() {
		return ((Integer) getAttrVal("Drainage"));
	}	
	public void setDrainage(Integer Drainage) {
		setAttrVal("Drainage", Drainage);
	}
	public String getObserilness() {
		return ((String) getAttrVal("Obserilness"));
	}	
	public void setObserilness(String Obserilness) {
		setAttrVal("Obserilness", Obserilness);
	}
	public String getId_rec_psn() {
		return ((String) getAttrVal("Id_rec_psn"));
	}	
	public void setId_rec_psn(String Id_rec_psn) {
		setAttrVal("Id_rec_psn", Id_rec_psn);
	}
	public String getCreatedby() {
		return ((String) getAttrVal("Createdby"));
	}	
	public void setCreatedby(String Createdby) {
		setAttrVal("Createdby", Createdby);
	}
	public FDateTime getCreatedtime() {
		return ((FDateTime) getAttrVal("Createdtime"));
	}	
	public void setCreatedtime(FDateTime Createdtime) {
		setAttrVal("Createdtime", Createdtime);
	}
	public String getModifiedby() {
		return ((String) getAttrVal("Modifiedby"));
	}	
	public void setModifiedby(String Modifiedby) {
		setAttrVal("Modifiedby", Modifiedby);
	}
	public FDateTime getModifiedtime() {
		return ((FDateTime) getAttrVal("Modifiedtime"));
	}	
	public void setModifiedtime(FDateTime Modifiedtime) {
		setAttrVal("Modifiedtime", Modifiedtime);
	}
	public String getName_bleed() {
		return ((String) getAttrVal("Name_bleed"));
	}	
	public void setName_bleed(String Name_bleed) {
		setAttrVal("Name_bleed", Name_bleed);
	}
	public String getName_rec_psn() {
		return ((String) getAttrVal("Name_rec_psn"));
	}	
	public void setName_rec_psn(String Name_rec_psn) {
		setAttrVal("Name_rec_psn", Name_rec_psn);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id_rec";
	}
	
	@Override
	public String getTableName() {	  
		return "CI_MR_NU_GYMISSABORTREC";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(GyMissAbortRecDODesc.class);
	}
	
}